api/constants/constants.go

```diff
@@ -431,6 +431,8 @@ const (
 	// long-lived connections alive as L7 LB usually ignores TCP keepalives and
 	// has very short idle timeouts.
 	WebAPIConnUpgradeTypeALPNPing = "alpn-ping"
+	// WebAPIConnUpgradeTypeWebSocket is the standard upgrade type for WebSocket.
```

Comment:
Perhaps I'd link to the RFD here ? The client transport mechanisms can be confusing to those unfamiliar with them, and the RFD clears it up massively.

Reply:
I've added the rfd to this const block. Thanks =D